examples/common/mod.rs

```diff
@@ -10,20 +10,72 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  ********************************************************************************/
+use clap::Parser;
 use up_transport_zenoh::zenoh_config;
 
+#[derive(clap::ValueEnum, Clone, Copy, PartialEq, Eq, Hash, Debug)]
+pub enum Wai {
+    Peer,
+    Client,
+    Router,
+}
+
+#[derive(clap::Parser, Clone, PartialEq, Eq, Hash, Debug)]
+pub struct Args {
+    #[arg(short, long)]
+    /// A configuration file.
+    config: Option<String>,
+    #[arg(short, long)]
+    /// The Zenoh session mode [default: peer].
+    mode: Option<Wai>,
+    #[arg(short = 'e', long)]
+    /// Endpoints to connect to.
+    connect: Vec<String>,
+    #[arg(short, long)]
+    /// Endpoints to listen on.
+    listen: Vec<String>,
+    #[arg(long)]
+    /// Disable the multicast-based scouting mechanism.
+    no_multicast_scouting: bool,
+}
+
 #[allow(clippy::must_use_candidate, clippy::missing_panics_doc)]
 pub fn get_zenoh_config() -> zenoh_config::Config {
+    let args = Args::parse();
+
     // Load the config from file path
-    // Config Examples: https://github.com/eclipse-zenoh/zenoh/blob/0.10.1-rc/DEFAULT_CONFIG.json5
```

Comment:
I like providing an example config file. Maybe we can add one to this repo instead of pointing to an external one?

Reply:
Yes, I've added the config file.